app/controllers/people_controller.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+# crud for people. this is essentially the directory
+class PeopleController < ApplicationController
+  def index
+    authorize Person
+    @people = Person.all
+  end
+
+  def new
+    authorize Person
+    @person = Person.new
+  end
+
+  def create
+    authorize Person
+    @person = Person.new(person_params)
```

Comment:
I think a good pattern for actions with single records is to authorize on the RHS of the assignment. So, for this action and those below, it would be:


```suggestion
    @person = authorize Person.new(person_params)
```

This also ensures that policy predicates are respected in cases where there is some attribute dependent check. If we authorize before preparing the record with request params, we could inadvertantly miss some restriction by authorizing the vanilla record without whatever bonkers values the client may be passing.